utils/apiUtil.js

```diff
@@ -419,6 +419,22 @@ module.exports = {
           css: '.covid19-dashboard_counts',
         },
       },
+      accessclinical: {
+        summary: {
+          css: '.index-page_introduction',
+        },
+        cards: {
+          css: '.g3-button',
+        },
+      },
+      niaid: {
```

Comment:
we may want to set it to `qa-niaid` to avoid conflicts in other NIAID environments like niaiddata.org?

Reply:
good suggestion